app/models/service_template_ansible_playbook.rb

```diff
@@ -132,7 +132,7 @@ def self.validate_config_info(info)
 
 
   def job_template(action)
-    resource_actions.find_by(:action => action.to_s.capitalize).try(:configuration_template)
+    resource_actions.find_by(:action => action.capitalize).try(:configuration_template)
```

Comment:
I believe [this](https://github.com/bzwei/manageiq/blob/6d0ad8dbf7dc4e45e9af612bc15170ea19c2fd5d/app/models/service_template_ansible_playbook.rb#L149) will fail without the `to_s`.